intera_interface/src/intera_interface/limb.py

```diff
@@ -138,6 +160,12 @@ def __init__(self, limb="right", synchronous_pub=False):
             queue_size=1,
             tcp_nodelay=True)
 
+        ns_pkn = "ExternalTools/" + limb + "/PositionKinematicsNode/"
+        self._iksvc = rospy.ServiceProxy(ns_pkn + 'IKService', SolvePositionIK)
+        self._fksvc = rospy.ServiceProxy(ns_pkn + 'FKService', SolvePositionFK)
+        rospy.wait_for_service(ns_pkn + 'IKService', 5.0)
+        rospy.wait_for_service(ns_pkn + 'FKService', 5.0)
```

Comment:
That's what I'm about to work on. Both services will exist in `sawyer_simulator`, and certainly the FKService will exist in this release. I anticipate migrating to use `SNS` as the IK solver when creating the IKService